lib/on_click_cell_factory.rb

```diff
@@ -207,6 +211,22 @@ def run_operation_menu_item
     cmi
   end
 
+  def show_delete_menu_item
```

Comment:
You should add it to `enable_disable_conext_menu_items` as well, so that the context menu item is only enabled when actually a feed is selected.